infrastructure/quick-deploy/gcp/parameters.tfvars

```diff
@@ -245,6 +245,34 @@ keda = {
 
 mongodb = {
   node_selector = { service = "state-database" }
+
+  # Uncomment to define custom resources for MongoDB pods, comment for default values
+  mongodb_resources = {
+    limits = {
+      "cpu"               = 4
+      "memory"            = "8Gi"
+      "ephemeral-storage" = "20Gi"
+    }
+    requests = {
+      "cpu"               = 2
+      "memory"            = "4Gi"
+      "ephemeral-storage" = "2Gi"
+    }
+  }
```

Comment:
Why is this uncommented while it is commented on other deployments?

Reply:
I thought the default values from the source module were not fit for a cloud deployment.